docs/pages/components/date-picker.md

```diff
@@ -24,7 +24,7 @@ This component mostly relies on the CSS of other components and has very little
 			<div class="fd-input-group fd-input-group--after">
 				<input type="text" value="" placeholder="Pick a date">
 				<span class="fd-input-group__addon fd-input-group__addon--after fd-input-group__addon--button">
-				<button class="fd-popover__control fd-button--light sap-icon--calendar" aria-controls="Itcgq828" aria-expanded="false" aria-haspopup="true" aria-hidden="true"></button>
+				<button class="fd-button--light sap-icon--calendar" aria-controls="Itcgq828" aria-expanded="false" aria-haspopup="true" aria-hidden="true"></button>
```

Comment:
Can we remove this? I think this is needed for the implementation of popper.js in the fundamental-* libraries.